tensorboard/plugins/hparams/tf_hparams_utils/tf-hparams-utils.html

```diff
@@ -366,6 +373,20 @@
             [pathToArray, 0, currentArray.length].concat(newArray));
       }
       utils.setArrayObservably = setArrayObservably;
+
+      // Computes a simple not-secure 32-bit integer hash value for a string.
+      function stringHash(str) {
```

Comment:
I think the name tricked me. I assumed this returns a string hash of a data just looking at the name of the function and wrongly assumed you were doing arithmetic on string. 
Please let me know if it is just me.

Reply:
You're a good enough example. I renamed to hashOfString.